src/backend/distributed/utils/maintenanced.c

```diff
@@ -304,7 +304,12 @@ CitusMaintenanceDaemonMain(Datum main_arg)
 			 secondsSincePrevStatsCollection >= STATISTICS_COLLECTION_RETRY_INTERVAL))
 		{
 #if HAVE_LIBCURL
-			if (EnableStatisticsCollection)
+			bool citusHasBeenLoaded = false;
```

Comment:
Is there a strong reason not to move this all the way to the top of the `for` loop, such that if Citus is not loaded we simply `continue`?

Reply:
(i.e. is there a requirement that the maintenance loop do something even when Citus hasn't yet been loaded?)

Reply:
I couldn't find anything that we needed in maintenance loop when Citus is not loaded yet. Moved the check to top of the ```for``` loop as you suggested.